src/System.Collections/tests/Generic/SortedList/SortedList.Generic.Tests.cs

```diff
@@ -431,6 +431,20 @@ public void SortedList_Generic_RemoveAt_EveryValue(int count)
             }
         }
 
+        [Theory]
+        [MemberData("ValidCollectionSizes")]
```

Comment:
I'm not sure how this file escaped our replacing these with usage of nameof, but we should follow-up to fix that.  For now, it makes sense for you to remain consistent with the rest of the file and leave this as-is.